dnSpy/dnSpy/Documents/Tabs/DefaultDocumentList.cs

```diff
@@ -48,6 +50,81 @@ sealed class DefaultDocumentList {
 		public void Add(DsDocumentInfo file) => files.Add(file);
 	}
 
+	sealed class DotnetReferenceFileFinder {
+		readonly CancellationToken cancellationToken;
+		readonly List<DefaultDocumentList> allLists;
+
+		public DotnetReferenceFileFinder(CancellationToken cancellationToken) {
+			this.cancellationToken = cancellationToken;
+			allLists = new List<DefaultDocumentList>();
+		}
+
+		public IEnumerable<DefaultDocumentList> AllFiles => allLists.ToArray();
+
+		public void Find() {
+			cancellationToken.ThrowIfCancellationRequested();
+
+			var dotNetPathProvider = new DotNetPathProvider();
+
+			foreach (var frameworkPath in dotNetPathProvider.GetSharedDotNetPaths()) {
+				cancellationToken.ThrowIfCancellationRequested();
+				var appName = $"{Path.GetFileName(Path.GetDirectoryName(frameworkPath.Path))} {frameworkPath.Version} (x{frameworkPath.Bitness:d})";
+				allLists.Add(new DefaultDocumentList(appName, Directory.GetFiles(frameworkPath.Path, "*.dll").Where(FilterFiles).Select(DsDocumentInfo.CreateDocument)));
+			}
+		}
+
+		static bool FilterFiles(string file) {
+
+			var fileName = Path.GetFileName(file);
+
+			if (fileName.Length == 0) {
+				return false;
+			}
```

Comment:
```suggestion
			if (fileName.Length == 0)
				return false;
```

Use `if` statements without braces when the body of the `if` is a single line.